SocialFilter/sections/general_extensions.txt

```diff
@@ -69,6 +69,7 @@ warezcrack.net#$#.onp-sl-content > [data-lock-id^="onpLock"] { display: block!im
 0lik.ru##.onp-sl-outer-wrap
 0lik.ru$replace=/locker:{\s+timer:\s'\d\d'/locker: { timer: '1'/
 !
+sports.khan.co.kr#$##replyBalloon { margin-left: 0px !important; }

```

Comment:
No need to do it. Better block whole widget.

Reply:
OK.